app/pages/Covid/Covid.tsx

```diff
@@ -77,13 +77,11 @@ const CovidRoutes = () => {
   );
 };
 
-const CovidPages = () => (
+export const CovidPages = () => (
```

Comment:
My usual objection to it in large projects, it encourages renaming components when you import them, which is a pain to trace and debug.

But not militant about it, either way we should pick one convention in the whole codebase and work towards it. It's pretty inconsistent ATM.

Reply:
> My usual objection to it in large projects, it encourages renaming components when you import them, which is a pain to trace and debug.

Fair point, although I think most editors have powerful enough static analyzers to resolve the imported name to the original definition, even if they mismatch. I say this as vim user who somewhat recently started using language servers.

But I'm also not militant and would prefer consistency over anything else. Maybe we can go with always using named exports, but also having the convention that the "main" export should have the same name as the file? This specific line is a bit annoying because in order to import it, you have to remember that the file name is `Covid` but that the named import is `CovidPages`.

Reply:
Yep 100% agree on naming. I wrote a simple custom eslint rule ages ago to enforce that `file-name-matches-class-name` that I use in most of my personal projects with service classes, can probably adapt to enforce it for functional react components easily.

For now, will rename the files I've converted to typescript and can adapt the others as we go along. `Covid` -> `CovidPages` - Though for consistency, would probably go for `CovidPage` or `CovidInfoPage`)